### PR TITLE
Update track cuts for pass3 PbPb analyses

### PR DESCRIPTION
**Pull request by fgrosa**

### CODE REVIEW

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C [hunk 1]
```diff
@@ -956,6 +964,7 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs3050_Central2018_Pass3(bool fUseStrongPID
     esdTrackCuts->SetRequireTPCRefit(true);
     esdTrackCuts->SetRequireITSRefit(true);
     esdTrackCuts->SetMaxChi2PerClusterTPC(2.5);
+    esdTrackCuts->SetMinNCrossedRowsTPC(70);
```
Comment (fcatalan92) [inline cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C:967]: Isn't this cut applied in the filtering?

**Review by fcatalan92 — CHANGES_REQUESTED** [review]
I would not modify the old configurations to keep track of what was used in the past. For the config files of Raa and v2 in 30-50, I have created new functions for pass3 so there is no need to update the others.

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C (hunk 1 above)
Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C:967]: It should be in the FB4, but I could not find the list of cuts in the FB4, so to be 100% sure I added it (it does not hurt in any case)

Comment (fcatalan92) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C:967]: It is in FB4 https://twiki.cern.ch/twiki/pub/ALICE/PWGPPAODTrackCuts/rs_aodbits_130214_copy.pdf, https://github.com/alisw/AliRoot/blob/master/ANALYSIS/ANALYSISalice/AliESDtrackCuts.cxx#L821

Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_3050_Raa.C]: Ciao Fabio, thanks a lot for checking! I have remove it and I reverted the mtehods for the Ds in pass1 for bookkeeping as you suggested (I did not do it for the D+, which we mainly use for checks)

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_010_Raa.C [hunk 2]
```diff
@@ -942,6 +946,7 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs010_FiltTreeCreator2018QM(bool fIsMC=false
     analysiscuts->SetPtBins(nptbins+1,ptbins);
     analysiscuts->SetCuts(nvars,nptbins,anacutsval);
     analysiscuts->AddTrackCuts(esdTrackCuts);
+    analysiscuts->SetMinNumTPCClsForPID(50);
```
Comment (fcatalan92) [inline cutobjects/PbPb/CutsDstoKKpi_PbPb2018_010_Raa.C:949]: Leftover

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C [hunk 3]
```diff
@@ -23,6 +23,8 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs6080_Central2018(bool fUseStrongPID = true
     esdTrackCuts->SetRequireSigmaToVertex(false);
     esdTrackCuts->SetRequireTPCRefit(true);
     esdTrackCuts->SetRequireITSRefit(true);
+    esdTrackCuts->SetMaxChi2PerClusterTPC(2.5);
+    esdTrackCuts->SetMinNCrossedRowsTPC(70);
```
Comment (fcatalan92) [inline cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:27]: Leftover

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C [hunk 4]
```diff
@@ -282,6 +285,8 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs6080_Loose2018(bool fUseStrongPID = true,
...
```
Comment (fcatalan92) [inline cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:289]: Leftover

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C [hunk 5]
```diff
@@ -515,6 +521,8 @@ AliRDHFCutsDstoKKpi* MakeFileForCutsDs6080_FiltTreeCreator2018(bool fUseStrongPI
...
```
Comment (fcatalan92) [inline cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:525]: Leftover

**Review by fcatalan92 — APPROVED** [review]

**Comment by fcatalan92** [comment]
Please remove some leftovers. The rest is fine

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C (hunk 5 above)
Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:525]: FIxed

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C (hunk 4 above)
Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:289]: Fixed

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C (hunk 3 above)
Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:27]: FIxed

Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:27]: Fixed

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C (hunk 5 above)
Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_6080_Raa.C:525]: Fixed

File: cutobjects/PbPb/CutsDstoKKpi_PbPb2018_010_Raa.C (hunk 2 above)
Comment (fgrosa) [reply cutobjects/PbPb/CutsDstoKKpi_PbPb2018_010_Raa.C]: Fixed

**Pull request merged**